js/src/utils.ts

```diff
@@ -34,3 +38,68 @@ export async function replaceQrCode(data: string, templateString: string): Promi
         return templateString;
     }
 }
+
+
+export function replaceBenefits(jsonObject: any, svgTemplate: string): string {
+    try {
+        const credentialSubject = jsonObject.credentialSubject;
+        const benefitsArray = credentialSubject.benefits as string[];
+
+        const benefitsString = benefitsArray.join(',');
+        const benefitsPlaceholderList = [BENEFITS_PLACEHOLDER1, BENEFITS_PLACEHOLDER2]
+        const replacedSvgWithBenefits = replaceMultiLinePlaceholders(svgTemplate,benefitsString,35, benefitsPlaceholderList)
+        return replacedSvgWithBenefits;
+    } catch (e) {
+        console.error(e);
+        return svgTemplate;
+    }
+}
+
+export function replaceAddress(jsonObject: any, svgTemplate: string): string {
+    try {
+      const credentialSubject = jsonObject.credentialSubject;
+      
+      const fields = ["addressLine1", "addressLine2", "addressLine3", "city", "province", "region", "postalCode"];
```

Comment:
Are these predefined constant values? If yes Can we move this into a top level constant file than within the mehtod?

Reply:
Moved to constants